controllers/ovncontroller_controller.go

```diff
@@ -547,6 +558,8 @@ func (r *OVNControllerReconciler) generateServiceConfigMaps(
 	templateParameters := make(map[string]interface{})
 	if instance.Spec.NetworkAttachment != "" {
 		templateParameters["OvnEncapNIC"] = nad.GetNetworkIFName(instance.Spec.NetworkAttachment)
+	} else if contains(instance.Spec.NetworkAttachments, "tenant") {
```

Comment:
Should we give preference to the "new" way (first branch for NADs, second - for NAD)?